powersimdata/scenario/analyze.py

```diff
@@ -323,6 +323,36 @@ def get_wind(self):
         profile = TransformProfile(self._scenario_info, self.get_grid(), self.get_ct())
         return profile.get_profile("wind")
 
+
+    def get_wind_onshore(self):
+        """Returns wind onshore profile
+ 
+        :return: (*pandas.DataFrame*) -- data frame of wind energy output for onshore
+            turbines
+        """
+        profile = TransformProfile(self._scenario_info, self.get_grid(), self.get_ct())
+        wind = profile.get_profile("wind")
+ 
+        grid = self.get_grid()
+        onshore_id = grid.plant.groupby(["type"]).get_group("wind").index
+        return wind[onshore_id]
+
+    def get_wind_offshore(self):
+        """Returns wind offshore profile
+ 
+        :return: (*pandas.DataFrame*) -- data frame of wind energy output for offshore
+            turbines
```

Comment:
Missing entry in docstring for the raised error. You should add: `:raises ValueError: if no offshore wind turbinee is in the grid`

Reply:
done